app/controllers/albums_controller.rb

```diff
@@ -0,0 +1,52 @@
+class AlbumsController < ApplicationController
+
+  def index
+    @albums = Album.all
+  end
+
+  def new
+    @album = Album.new
+  end
+
+  def create
+    @album = Album.new(album_params)
+    if @album.save
+      redirect_to action: 'index'
+    else
+      render :new
+    end
+  end
+
+  def show
+    @album = get_album
```

Comment:
yes.

some people use controller `before_action`s to load in variables, but that is an antipattern to me because it obfuscates what is going on